Makefile

```
@@ -500,6 +502,11 @@ verify-gen: generate
 		echo "generated files are out of date, run make generate"; exit 1; \
 	fi
 
+.PHONY: capd-verify
+capd-verify:
```

```suggestion
verify-docker-provider:
```

for consistency with the other targets